### ISSUE TITLE
Should categories really fallback to /sl if no /tl?

### ISSUE THREAD

**Issue by unhammer** [apertium/apertium-recursive#79]
If we have something like

```
number = (GD sg) sg pl GD ;
a_adj = sint ord ;
adj: _.a_adj.number;
A -> %adj { %1 };
```
and bidix output is `^good<adj><sint>/OK<adj>$` then I don't want the `<sint>` to be output, that's an "inherent" tag (the word `OK` is can't do synthetic comparative/superlative, so it doesn't get a `<sint>`). But to ensure that, I have to explicitly say that the attribute should have the tl value, for each such where bidix can remove it:
```
A -> %adj { %1[a_adj=1.a_adj/tl]  };
```

I know I can always be explicit like this, but I would think the default should be to not fallback here, that if bidix removes a tag, it's for a reason. Does anyone actively use sl fallbacks? 

@jonorthwash @ftyers @MarcRiera and anyone else using apertium-recursive

**Comment by jonorthwash** [comment]
Yes, I've run into this before and agree.  Subcategory `sl` or `tl` tags (or even grammatical tags) included in bidix should somehow be treated as a chunks by default.  I've talked with @mr-martian about it before, and it seems like there are two issues involved in its implementation:
- What is the right way to  handle all this, at the level of the syntax?
- How to implement it technically?

My impression is that the first issue is the blocking one.

**Comment by mr-martian** [comment]
A couple of options that come to mind that might address this issue and definitely wouldn't break backwards compatibility:

`adj: _.a_adj/tl.number` so that `a_adj` will never get clipped from `sl` but `number` retains the current default behavior

A compiler option that makes all unspecified clips be `tl` clips rather than the current `tl > ref > sl` fallback approach

Either of those would be pretty easy to implement, but there could very well be an option I haven't thought of that would be more correct.

**Comment by unhammer** [comment]
I would prefer an all-or-nothing option, but would it be possible to do it as a file pragma? Like the [CG subreadings](https://visl.sdu.dk/cg3/single/#sub-stream-apertium) option, we could have `SIDE_SOURCES = tl ; ` or something (default value being `= tl ref sl ;`).

Since the rules in the files necessarily will depend on the value of that option, I prefer having it inside the rtx, instead of having to look it up in Makefile.am. (Also, having to remember compile switches while doing rtx-comp manually to debug means one more way I can mess things up at the time when I am the most confused.)

**Comment by MarcRiera** [comment]
I've just bumped into this while doing some tests with `apertium-spa-cat`, where some entries "lose" a tag in bidix.

I agree with @unhammer regarding including the option directly in the file. Using a compiler option will probably lead to confusion. However, I think the two proposed solutions shouldn't be mutually exclusive. Currently I don't really need fallbacks and disabling them entirely would solve my issues, but I can imagine myself using them in the future. In that case, it would be good to have the ability to do something like `adj: _.a_adj/tl.number` and keep fallbacks enabled globally. Otherwise, we would need to add conditionals and rules could get too complicated.
